src/Resources/contao/library/Contao/Controller.php

```diff
@@ -1506,8 +1506,14 @@ public static function addImageToTemplate($objTemplate, $arrItem, $intMaxWidth=n
 
 		try
 		{
-			$src = \Image::create($arrItem['singleSRC'], $size)->executeResize()->getResizedPath();
-			$picture = \Picture::create($arrItem['singleSRC'], $size)->getTemplateData();
+			$src = \System::getContainer()->get('contao.image.image_factory')->create(TL_ROOT . '/' . $arrItem['singleSRC'], $size)->getUrl(TL_ROOT);
```

Comment:
We are passing an absolute path (`TL_ROOT . '/' . $arrItem['singleSrc']`) to the `create()` method to strip it again in the `getUrl()` method?